buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -134,6 +136,13 @@ class ClusterFormationTasks {
                         esConfig['discovery.zen.hosts_provider'] = 'file'
                     }
                     esConfig['discovery.zen.ping.unicast.hosts'] = []
+                    if (esConfig.containsKey('discovery.type') == false
+                      && config.minimumMasterNodes.call() > 1 && hasBwcNodes == false) {
```

Comment:
Could this condition just be `hasBwcNodes == false`?

Reply:
@DaveCTurner I need the first line to have my override of the type to `zen.

But yea the minimum master node check can go away now that the setting of the initial nodes list is there I think, trying it now.

Reply:
Removing the master node count causes at least the docs tests for https://github.com/elastic/elasticsearch/blob/master/docs/reference/modules/node.asciidoc to fail  (rightfully so for zen2 I guess).

Want to fix that here or in a follow up?

Reply:
This test is now skipped :)